lib/services.js

```diff
@@ -128,6 +130,26 @@ const services = {
         if (headers && headers['x-amz-website-redirect-location']) {
             md.setRedirectLocation(headers['x-amz-website-redirect-location']);
         }
+        // Stores retention information if object either has its own retention
+        // configuration or default retention configuration from its bucket
+        const headerMode = headers['x-amz-object-lock-mode'];
```

Comment:
could you pull this logic (evaluating headers vs defaultRetention) into another method in `objectLockHelpers`? then we'll be able to use it for the intitiateMPU update :)

Reply:
sure thing :)

Reply:
@dora-korpar done; now it's a separate method that can be reused.